crates/loam-cli/src/commands/dev/mod.rs

```diff
@@ -0,0 +1,208 @@
+                            // Ignore temporary files
+                            if is_temporary_file(path) {
+                                return;
+                            }
+                            let env_toml_parent_abs = canonicalize_path(&env_toml_parent_clone);
+                            let env_toml_path_abs = canonicalize_path(&env_toml_path_clone);
+                            let parent_is_env_toml_parent =
+                                path.parent() == Some(env_toml_parent_abs.as_path());
+                            let path_is_env_toml = path == env_toml_path_abs.as_path();
+                            let parent_is_in_watched_dirs = is_parent_in_watched_dirs(
+                                &env_toml_parent_abs,
+                                &watched_dirs_clone,
+                            );
+                            // skip if the file is in the parent directory of environments.toml and it is not environments.toml
+                            if !(parent_is_env_toml_parent
+                                && !path_is_env_toml
+                                && !parent_is_in_watched_dirs)
+                            {
+                                println!("File changed: {path:?}");
+                                if let Err(e) = tx.blocking_send(Message::FileChanged) {
+                                    eprintln!("Error sending through channel: {e}");
+                                }
+                            }
+                        }
+                    }
+                }
+            })
+            .unwrap();
+        // Watch the parent directory of environments.toml
+        watcher.watch(env_toml_parent.as_path(), RecursiveMode::NonRecursive)?;
+        for package_path in watched_dirs {
+            watcher.watch(package_path.as_path(), RecursiveMode::Recursive)?;
+        }
+
+        if let Err(e) = self.build().await {
+            eprintln!("Build error: {e}");
+        }
+        println!("Watching for changes. Press Ctrl+C to stop.");
+
+        let rebuild_state_clone = Arc::clone(&rebuild_state);
+        loop {
+            tokio::select! {
+                _ = rx.recv() => {
+                    let mut state = rebuild_state_clone.lock().await;
+                    if !state.pending {
+                        state.pending = true;
+                        tokio::spawn(self.clone().debounced_rebuild(Arc::clone(&rebuild_state_clone)));
+                    }
+                }
+                _ = tokio::signal::ctrl_c() => {
+                    println!("Stopping dev mode.");
+                    break;
+                }
+            }
+        }
+        Ok(())
+    }
+
+    async fn debounced_rebuild(self, rebuild_state: Arc<Mutex<RebuildState>>) {
+        // Debounce to avoid multiple rapid rebuilds
+        time::sleep(std::time::Duration::from_secs(1)).await;
+
+        println!("Changes detected. Rebuilding...");
```

Comment:
```suggestion
        eprintln!("Changes detected. Rebuilding...");
```

Reply:
should all the println statements be eprintln?